courses/factories.py

```diff
@@ -74,6 +84,7 @@ class CourseRunEnrollmentFactory(DjangoModelFactory):
 
     user = SubFactory(UserFactory)
     run = SubFactory(CourseRunFactory)
+    company = SubFactory(CompanyFactory)
```

Comment:
Suggest you change this to reflect the data model more accurately and so we can ensure we don't accidentally avoid testing both cases:

```python
company = factory.Maybe(
    factory.fFaker('pybool'),
    yes_delcaration=SubFactory(CompanyFactory),
    no_declaration=None,
)
```

Then later so you can `CourseRunEnrollmentFactory.create(has_company_affiliation=True)` where you need to guarantee it:
```python
class Params:
    has_company_affiliation= Trait(company=SubFactory(CompanyFactory))
```